latch_cli/services/init/init.py

```diff
@@ -110,7 +110,11 @@ def _gen_assemble_and_sort(pkg_root: Path):
         shell=True,
     )
     subprocess.run(
-        f"rm -r {str(pkg_root / bowtie2_base_name)} {str(pkg_root / bowtie2_base_name)}.zip {str(pkg_root / 'bowtie2')}/*-debug*",
+        (
+            "rm -r"
+            f" {str(pkg_root / bowtie2_base_name)} {str(pkg_root / bowtie2_base_name)}.zip"
+            f" {str(pkg_root / 'bowtie2')}/*-debug*"
+        ),
```

Comment:
use `Path.unlink()` for the non-glob paths and use `glob.glob` to iterate through and unlink the globbed paths